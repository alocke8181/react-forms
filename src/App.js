import logo from './logo.svg';
import './App.css';
import BoxList from './BoxList';
import ToDoList from './ToDoList';

function App() {
  return (
    <div>
      <BoxList />
      <hr></hr>
      <ToDoList />
    </div>
  );
}

export default App;
