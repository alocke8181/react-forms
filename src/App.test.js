import { render } from '@testing-library/react';
import App from './App';

test('smoke test', () => {
  render(<App />);
});

test('snapshot', ()=>{
  const {asFragment} = render(<App />);
  expect(asFragment()).toMatchSnapshot();
})
