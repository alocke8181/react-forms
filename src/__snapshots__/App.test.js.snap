// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <form>
        <label
          for="bgColor"
        >
          Background Color
        </label>
        <input
          id="bgColor"
          name="bgColor"
          placeholder="#000000"
          type="color"
          value="#000000"
        />
        <label
          for="width"
        >
          Width px
        </label>
        <input
          id="width"
          name="width"
          placeholder="100"
          type="number"
          value="0"
        />
        <label
          for="height"
        >
          Height px
        </label>
        <input
          id="height"
          name="height"
          placeholder="100"
          type="number"
          value="0"
        />
        <button
          id="box-submit"
        >
          Submit
        </button>
      </form>
      <div />
    </div>
    <hr />
    <div>
      <form>
        <label
          for="text"
        >
          New Todo Item:
        </label>
        <input
          id="text"
          name="text"
          placeholder=""
          type="text"
          value=""
        />
        <button
          id="todo-submit"
        >
          Submit
        </button>
      </form>
      <div>
        <ul />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
